bool query
---
string color
float32[] abscissa
float32[] intensities
float32[] diffs
float32[] markersH
float32[] markersV
